src/components/Section.js

```diff
@@ -49,7 +57,7 @@ export default class Section extends React.Component {
 
   render() {
     const {section} = this.props;
-    const collapsed = this.state.collapsed && (section.type !== "introduction");
+    const collapsed = this.state.collapsed && !["introduction", "closure info"].includes(section.type);
```

Comment:
The `!["introduction", "closure info"].includes(section.type)` fragment is now duplicated twice... maybe move it into a helper fn?